### ISSUE TITLE
Frames of input are dropped frequently

### ISSUE THREAD

**Issue by charasyn** [charasyn/ps2tosnes#1]
It's hard to notice when you're just playing Mario Paint, but I determined that about 1-2 times a second, frames of input are dropped. This is most likely due to the other Arduino support software running in the interrupt vector and after loop(). I have decided to rewrite this to never return from loop and to never enable interrupts, bypassing most of the Arduino code with direct register operations instead. I expect that this should fix the sync issues.

**Comment by charasyn** [comment]
Just an update: I have yet to rewrite it, but I expect it shouldn't be too hard to do considering the recent updates to pc2snes.

**Comment by DobaMuffin** [comment]
Hey cooprocks123e, did you ever fix this issue?

**Comment by charasyn** [comment]
No. This project has been abandoned. I don't think it would be that hard to
figure out, however.
